log/log.go

```diff
@@ -3,25 +3,30 @@ package log
 import (
 	"strings"
 
+	viperutil "github.com/Conflux-Chain/go-conflux-util/viper"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"github.com/spf13/viper"
 )
 
-// InitFromViper inits logging from viper settings and adapts Geth logger.
+// LoggingConfig logging configuration such as log level etc.,
+type LoggingConfig struct {
+	Level string `default:"info"` // logging level
+}
+
+// MustInit inits logging from viper settings and adapts Geth logger.
 //
-// Note that viper must be initilized before, otherwise settings will not
-// be loaded correctly.
-func InitFromViper() {
-	lvl := viper.GetString("log.level")
-	level, err := logrus.ParseLevel(lvl)
-	if err != nil {
-		logrus.WithField("logLevel", lvl).WithError(err).Info(
-			"Invalid log level parsed from viper config, use info level as default.",
-		)
+// Note that viper must be initilized before this, and it will panic
+// and exit if any error happens.
+func MustInit() {
```

Comment:
Please rename to `MustInitFromViper` to aware about viper.

Reply:
done